x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/esnative/ReservedRealmTests.java

```diff
@@ -6,6 +6,7 @@
  */
 package org.elasticsearch.xpack.security.authc.esnative;
 
+import org.elasticsearch.ElasticsearchAuthenticationProcessingError;
```

Comment:
I think I marked all places here but we need to ensure that our passwords are long enough so that these tests don't fail in fips mode ( or explicitly use PBKDF2_STRETCH, but the former feels easier )